src/app/components/Image/index.stories.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Image from './index';
+
+const imageData = arrayOfBlocks => ({
+  blockId: '',
```

Comment:
It looks like the blockIds might be used for something but they're not, any reason you left them in? We can remove them for succinctness otherwise.

This applies to all other properties that aren't uses as well - there are some in the test file. I think it is just clearer for anyone refactoring this component that the stories and tests around it only include data that is relevant to that specific component.

That would be unattainable when we move to static fixture data in the tests and stories, but I think it is somewhat valuable now while we are being explicit.

Reply:
I agree with removing them from the `stories` file - however, I think the `test` file should use full fixture data. This is because it enables us to trust the test files. When we add new features to components, we only need to make the tests pass against the existing tests.